scripts/apothecary/formulas/openSSL/buildwin.cmd

```diff
@@ -0,0 +1,46 @@
+@echo off
```

Comment:
Yep, I see what you say about cmd just after my comment, and you're totally right.

About binaries, I've seen on OpenSSL site another binary compilation site (https://www.openssl.org/about/binaries.html):

> http://indy.fulgan.com/SSL/
   Pre-compiled Win32/64 libraries without external dependencies to the Microsoft Visual Studio Runtime DLLs, expect for the system provided msvcrt.dll

But it seems down currently...